index.html

```diff
@@ -232,12 +242,13 @@ <h3 class="mainText CreteRoundfont">About me</h3>
     </section>
 
     <footer class="footer">
-      <p class="Interfont footerTxt">
+      <div class="footer_contact">
+      <p class="item_1 Interfont footerTxt">
         I'm always interested in hearing about new projects, so if you'd like to
         chat please get in touch.
       </p>
       <form
-        class="contactForm"
+        class="contactForm item_2"
         action="https://formspree.io/f/xeqnpapb"
         method="POST"
```

Comment:
- [ ] **[OPTIONAL]**
- Although the previous reviewer didn't point this out, however, consider them as **MANDATORY REQUIREMENT** for this project, try addressing them in your spare time..

- Kindly reduce the width of your paragraph and align the button to the left on the form
**YOUR DESIGN** :point_down: 

![FORM](https://user-images.githubusercontent.com/59397159/167248070-5b528a6c-2f92-4600-a01a-f9057783c2f3.png)


**FIGMA DESIGN** :point_down: 

![FORMS](https://user-images.githubusercontent.com/59397159/167248092-e12da1c2-6a8f-4e14-9fc2-f2cbecc76531.png)